server/src/completions/overrides.ts

```diff
@@ -0,0 +1,20 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) 2023 Savoir-faire Linux. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+export const OVERRIDES = [
+  'append',
+  'prepend',
+  'remove',
+  'os',
+  'nooverride',
+  'qemuarm',
```

Comment:
the following overrides are specific to this yocto project. Any other override can be defined by the user. There are also special overrides like task-* or pn-* that apply to specific things. If this list is considered the final list of overrides then it is not sufficient. If it's just a list for a set of tested or documented ones in poky it's alright.

Reply:
Removed.